src/mbgl/gl/context.cpp

```diff
@@ -3,26 +3,29 @@
 #include <mbgl/gfx/shader_registry.hpp>
 #include <mbgl/gl/command_encoder.hpp>
 #include <mbgl/gl/defines.hpp>
-#include <mbgl/gl/drawable_gl.hpp>
-#include <mbgl/gl/drawable_gl_builder.hpp>
-#include <mbgl/gl/drawable_gl_tweaker.hpp>
 #include <mbgl/gl/draw_scope_resource.hpp>
 #include <mbgl/gl/enum.hpp>
-#include <mbgl/gl/layer_group_gl.hpp>
 #include <mbgl/gl/renderer_backend.hpp>
 #include <mbgl/gl/renderbuffer_resource.hpp>
-#include <mbgl/gl/uniform_buffer_gl.hpp>
 #include <mbgl/gl/texture_resource.hpp>
 #include <mbgl/gl/texture.hpp>
-#include <mbgl/gl/texture2d.hpp>
 #include <mbgl/gl/offscreen_texture.hpp>
 #include <mbgl/gl/debugging_extension.hpp>
 #include <mbgl/renderer/paint_parameters.hpp>
-#include <mbgl/shaders/gl/shader_program_gl.hpp>
 #include <mbgl/util/traits.hpp>
 #include <mbgl/util/std.hpp>
 #include <mbgl/util/logging.hpp>
 
+#if MLN_DRAWABLE_RENDERER
+#include <mbgl/gl/drawable_gl.hpp>
+#include <mbgl/gl/drawable_gl_builder.hpp>
+#include <mbgl/gl/drawable_gl_tweaker.hpp>
+#include <mbgl/gl/layer_group_gl.hpp>
+#include <mbgl/gl/uniform_buffer_gl.hpp>
+#include <mbgl/gl/texture2d.hpp>
+#include <mbgl/shaders/gl/shader_program_gl.hpp>
+#endif
+
 #include <cstring>
 #include <iterator>
 
```

Comment:
I am not sure how this is related but I get a crash in `Context::initializeExtensions `at `getProcAddress` line 87.
`probe.second` is _"glPushGroupMarkerEXT"_
Env: Mac, iOS Simulator (iPad), Debug build